app/Entirety/devices/templates/devices/detail.html

```diff
@@ -1,4 +1,6 @@
 {% extends '_base.html' %}
+{% load crispy_forms_tags %}
+
 
 {% block title %}
     {% if action == "Create" %}
```

Comment:
Somehow there is a blank space between Top bar and title.
![image](https://github.com/N5GEH/n5geh.tools.entirety/assets/79838286/2b95715c-1755-44b1-a098-6a6402280169)

Reply:
It should work now